domains/sh770

```diff
@@ -0,0 +1,12 @@
+{
+    "owner": {
+        "username": "sh770",
+        "email": "[sh770h@gmail.com]"
+       
+    },
+    "record": {
+        "CNAME": "sh770.github.io/dev"
+        "TXT": "_github-pages-challenge-sh770.sh770.dev"
+        "TXT": "f1bc37cc382372c5b5ef5103b14648"
+    }
+}
```

Comment:
Cannot have CNAME and  TXT records combined. See docs on how to do this: https://docs.is-a.dev/guides/github-pages/